File: Python/Beecrownd/Iniciantes/Tempo de Jogo com Minutos.py
hora_inicial, minuto_inicial, hora_final, minuto_final = map(int, input().split())
# Obs => o jogo tem duração mínima de um 1 minuto e maxima de 24 horas
# (1 minutos, 1440 minutos) Intervalo fechado

conversao_minuto_hora_inicial = hora_inicial*60
conversao_minuto_hora_final = hora_final*60
total_minutos_inicial = conversao_minuto_hora_inicial + minuto_inicial
total_minutos_final = conversao_minuto_hora_final + minuto_final
# total_minutos_tudo = total_minutos_inicial + total_minutos_final
# Horarios iguais 24 horas a duração
if hora_inicial == hora_final and minuto_inicial == minuto_final:
    print ("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
else:
    if total_minutos_inicial < total_minutos_final: # Não passa da meia noite
        diferenca_minutos = total_minutos_final - total_minutos_inicial
        horas = diferenca_minutos // 60
        minutos = diferenca_minutos % 60
        print (f"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)")
    else: # Passa da meia noite
        passou_da_meia_noite_ne_bb1 = (1440 - total_minutos_inicial) + total_minutos_final
        horas = passou_da_meia_noite_ne_bb1 // 60
        minutos = passou_da_meia_noite_ne_bb1 % 60
        print (f"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)")

File: Java/Beecrowd/Iniciantes/TempodeJogocomMinutos.java
// IntelliJ || NetBeans || Eclipse:

package desafiosbeecrowd;
import java.util.Locale;
import java.util.Scanner;
public class TempodeJogocomMinutos {
    public static void main (String [] args) {
        Locale.setDefault(Locale.US);
        Scanner scanner = new Scanner (System.in);
        int horaInicial = scanner.nextInt();
        int minutoInicial = scanner.nextInt();
        int horaFinal = scanner.nextInt();
        int minutoFinal = scanner.nextInt();

        // O ideal é converter tudo para uma unidade só:
        int conversaoMinutoHoraInicial = horaInicial*60;
        int conversaoMinutoHoraFinal = horaFinal*60;
        int totalMinutosInicial = conversaoMinutoHoraInicial + minutoInicial;
        int totalMinutosFinal = conversaoMinutoHoraFinal + minutoFinal;

        // Condicional caso o usuario digite as horas de começo e final iguais
        if (horaInicial == horaFinal && minutoInicial == minutoFinal) {
            System.out.println("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)");
        }
        else {
            // Caso o jogo não passe da meia noite
            if (totalMinutosInicial < totalMinutosFinal) {
                int diferencaMinutos = totalMinutosFinal - totalMinutosInicial;
                int horas = diferencaMinutos / 60; // Como eu to dividindo dois números inteiros o java pega apenas o inteiro a parte
                int minutos = diferencaMinutos % 60;
                System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horas, minutos);
            } else {
                int diferencaMinutos = (1440 - totalMinutosInicial) + totalMinutosFinal;
                int horas = diferencaMinutos / 60;
                int minutos = diferencaMinutos % 60;
                System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horas, minutos);
            }
        }
    }
}

// Terminal Beecrowd:

import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main (String []args) {
        Locale.setDefault(Locale.US);
        Scanner scanner = new Scanner (System.in);
        int horaInicial = scanner.nextInt();
        int minutoInicial = scanner.nextInt();
        int horaFinal = scanner.nextInt();
        int minutoFinal = scanner.nextInt();
        
        // O ideal é converter tudo para uma unidade só:
        int conversaoMinutoHoraInicial = horaInicial*60;
        int conversaoMinutoHoraFinal = horaFinal*60;
        int totalMinutosInicial = conversaoMinutoHoraInicial + minutoInicial;
        int totalMinutosFinal = conversaoMinutoHoraFinal + minutoFinal;
        
        // Condicional caso o usuario digite as horas de começo e final iguais
        if (horaInicial == horaFinal && minutoInicial == minutoFinal) {
            System.out.println("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)");
        }
        else {
            // Caso o jogo não passe da meia noite
            if (totalMinutosInicial < totalMinutosFinal) {
                int diferencaMinutos = totalMinutosFinal - totalMinutosInicial;
                int horas = diferencaMinutos / 60; // Como eu to dividindo dois números inteiros o java pega apenas o inteiro a parte
                int minutos = diferencaMinutos % 60;
                System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horas, minutos);
            }
            else {
                int diferencaMinutos = (1440 - totalMinutosInicial) + totalMinutosFinal;
                int horas = diferencaMinutos / 60;
                int minutos = diferencaMinutos % 60;
                System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horas, minutos);
            }
        }    
    }
}